workbench_utils.py

```diff
@@ -5067,45 +5067,45 @@ def validate_media_track_fields(config, csv_data):
                                         logging.error(message)
                                         sys.exit('Error: ' + message)
 
-                                # Confirm that config['media_use_tid'] and row-level media_use_term is for Service File (http://pcdm.org/use#ServiceFile).
-                                if 'media_use_tid' in row:
-                                    if row['media_use_tid'].startswith('http') and row['media_use_tid'] != 'http://pcdm.org/use#ServiceFile':
-                                        message = f"{row['media_use_tid']} cannot be used as a \"media_use_tid\" value in your CSV when creating media tracks."
-                                        logging.error(message)
-                                        sys.exit('Error: ' + message)
-                                    elif value_is_numeric(row['media_use_tid']):
-                                        media_use_uri = get_term_uri(config, row['media_use_tid'])
-                                        if media_use_uri != 'http://pcdm.org/use#ServiceFile':
```

Comment:
(@rosiel:) this is a messy git diff, because there are chunks of lines that appear the same at several levels of indent. 

Rather than what this looks like (big chunks of code deleted, and different ones added), all that was done was to indent a block of code four spaces.